src/shared/main/java/com/couchbase/lite/internal/support/Log.java

```diff
@@ -86,6 +88,15 @@ public static void initLogging(@NonNull Map<String, String> errorMessages) {
         Log.errorMessages = Collections.unmodifiableMap(errorMessages);
 
         Log.i(LogDomain.DATABASE, "Couchbase Lite initialized: " + CBLVersion.getVersionInfo());
+
+        final ExecutionService exec = CouchbaseLite.getExecutionService();
+        exec.postDelayedOnExecutor(
+            2 * 60 * 1000, // two minutes
+            exec.getMainExecutor(),
```

Comment:
This implementation is not the same as iOS and .NET. It might be a good idea to have the same implementation by checking this when creating a Database instance.

Reply:
I can do that if you wish.  It means that that, although the logger is created only once, in a given application, the check for logging behavior happens multiple times.  I claim that is a bug.

Reply:
.NET and iOS only notifies once, it was done under dispatch_once kind of thing.

Reply:
Ohhhh... The OS somehow tracks the job and refuses to run it more than once?  Mmmm... I don't know that there is any such thing in Android.  Lemme look...

Reply:
Modified to warn only once when
- File log level set from some other level, to NONE
- File logger config set from some other value to null
- The file logger is unconfigured when the first instance of Database is created.

Reply:
@Sandychuang8 @pasin : I believe I've addressed all of the issues.  Rereview?